simulators/portal-interop/src/main.rs

```diff
@@ -401,3 +412,28 @@ dyn_async! {
         }
     }
 }
+
+dyn_async! {
+    async fn test_find_nodes_zero_distance<'a>(test: &'a mut Test, client_a: Client, client_b: Client) {
+        let target_enr = match client_b.rpc.node_info().await {
+            Ok(node_info) => node_info.enr,
+            Err(err) => {
+                test.fatal(&format!("Error getting node info: {err:?}"));
+                return;
+            }
+        };
+
+        match client_a.rpc.find_nodes(target_enr.clone(), vec![0]).await {
+            Ok(response) => match response.enrs.get(0) {
```

Comment:
Additionally we could test if `response.enrs.len() == 1`

Reply:
resolved